src/User/Module.php

```diff
@@ -81,11 +81,19 @@ class Module extends BaseModule
      * @var bool whether to enable two factor authentication or not
      */
     public $enableTwoFactorAuthentication = false;
+    /**
+     * @var array list of channel for two factor authentication availables
+     */
+    public $twoFactorAuthenticationValidators = null;
     /**
      * @var int cycles of key generation are set on 30 sec. To avoid sync issues, increased validity up to 60 sec.
      * @see http://2fa-library.readthedocs.io/en/latest/
      */
     public $twoFactorAuthenticationCycles = 1;
+     /**
+     * @var int the number of seconds for which the code sent by email or by mobile phone is valid
+     * */
+    public $twoFactorEmailAuthenticationValidity = 300;
```

Comment:
Since the param is valid for both email and sms, it should have a more generic name, like `$twoFactorCodeValidityDuration`

Reply:
The param twoFactorEmailAuthenticationValidity is not more used. I deleted it